### ISSUE TITLE
Zero-Waste?

### ISSUE THREAD

**Issue by nelsonic** [dwyl/phase-two#96]
https://youtu.be/Gj38m08QLoE "_waste is just a failure of the imagination_"

Zero Waste Is Not the Only Solution - 4 Tips to Have a Bigger Impact: https://youtu.be/sO76q932VOo

How can we _not_ have a trash can [@home](https://github.com/dwyl/home)

**Comment by iteles** [comment]
Controlling your _own_ level of waste is a walk in the park compared to controlling the level of waste of 30 people. 

I certainly plan to have a feature in our app which tells people where and how to shop to reduce waste, we will have reuseable containers people can take shopping with them and we can also have and sell toiletries which we buy in bulk and people only need a small amount of (not a whole plastic bottle of) whilst they’re with us. 

These are only the things I’ve started thinking about but the real struggle will be food packaging.

**Comment by nelsonic** [comment]
@iteles indeed. _However_ the another way of looking at the challenge is that with 30 people we can buy most _seasonal_ veg+fruit in reasonable bulk and have it delivered in reusable crates. If we buy lentils, beans, rice, etc. in bulk too from distributors we can cut down a _lot_ of food packaging/waste. 💡

**Comment by nelsonic** [comment]
Why I live a zero waste life | Lauren Singer | TEDx
[![image](https://user-images.githubusercontent.com/194400/39895400-fcd8fd06-54a1-11e8-8ce0-5c6e1bf8e3d0.png)](https://youtu.be/pF72px2R3Hg)
http://trashisfortossers.com/my-waste-free-hair-care-routine

[![image](https://user-images.githubusercontent.com/194400/39898194-a054f146-54ad-11e8-8f4c-42bf5b4296bb.png)](https://youtu.be/CSUmo-40pqA)

related to #58 
https://news.nationalgeographic.com/2018/05/plastic-bag-mariana-trench-pollution-science-spd 😞
